html/form/basic.php

```diff
@@ -1,16 +1,18 @@
-<form id="smly" class="container" action="#" method="post">
-	<p class="error" style="padding:15px;background-color:#f2dede;margin:0 0 10px;display:none"></p>
-	<p class="success" style="padding:15px;background-color:#dff0d8;margin:0 0 10px;display:none"><?php echo esc_html__( 'Thank you for subscribing to our newsletter.', 'wp_sendsmaily' ); ?></p>
+<form id="smly" class="container" action="<?php echo admin_url('admin-post.php'); ?>" method="post">
+	<?php if ( $this->api_credentials === '' ) { ?>
+		<p class="no_auth" style="padding:15px;background-color:#f2dede;margin:0 0 10px"><?php echo esc_html__( 'Smaily credentials not validated. Subscription form will not work!', 'wp_smaily'); ?></p>
+	<?php } ?>
```

Comment:
You can remove this if statement. Error is already handled by `Smaily_Newsletter_Subscription_Widget::widget`.